packages-exp/remote-config-compat/src/remoteConfig.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '@firebase/app-types';
+import { RemoteConfig } from '@firebase/remote-config-types-exp';
+import {
+  Value as ValueCompat,
+  FetchStatus as FetchSTatusCompat,
+  Settings as SettingsCompat,
+  LogLevel as RemoteConfigLogLevel,
+  RemoteConfig as RemoteConfigCompat
+} from '@firebase/remote-config-types';
+import {
+  setLogLevel,
+  activate,
+  ensureInitialized,
+  fetchAndActivate,
+  fetchConfig,
+  getAll,
+  getBoolean,
+  getNumber,
+  getString,
+  getValue
+} from '@firebase/remote-config-exp';
+import { FirebaseService } from '@firebase/app-types/private';
+
+export class RemoteConfigCompatImpl
+  implements RemoteConfigCompat, FirebaseService {
+  constructor(public app: FirebaseApp, private _remoteConfig: RemoteConfig) {}
+
+  get defaultConfig(): { [key: string]: string | number | boolean } {
+    return this._remoteConfig.defaultConfig;
```

Comment:
Thinking: the plan to modularize attribute accessors is to have ppl call `getRemoteConfig()` and then calling the accessor on the returned `RemoteConfig` instance.